.gitignore

```diff
@@ -0,0 +1,134 @@
+# Logs
```

Comment:
Pretty much the entire contents of this file can be removed. Only files/folders that are relevant should be included in here, in this case the `node_modules` directory, and the `.env` & `.env.test` files. The rest of it is boilerplate stuff for other applications that have nothing to do with this project